internal/rollup/rollup_bundle.bzl

```diff
@@ -169,7 +169,10 @@ def _run_rollup(ctx, sources, config, output, map_output = None):
 
     # We will produce errors as needed. Anything else is spammy: a well-behaved
     # bazel rule prints nothing on success.
-    args.add("--silent")
+    #
+    # TODO(https://github.com/rollup/rollup/issues/2582): Uncomment the following
+    # line once rollup stops reporting build errors with onwarn.
+    # args.add("--silent")
```

Comment:
my concern is what does an execution of bazel look like now? Does rollup print a bunch of stuff to the terminal? as said in the original comment, a well-behaved bazel rule prints nothing on success. Can we somehow throw away the stdout from rollup if the compilation is successful?

Reply:
Totally understand. However, in this case I'd prioritize the correctness over the noise made by rollup. OTOH, this is actually inevitable in the nodejs world, even if the rollup is silenced, some poorly written plugin may console.log something from nowhere . So I think we can replace this comment with a more descriptive message pointing to a bug in rules_nodejs saying we should fix the log spam in the future. What do you think?